<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VnosObveznosti</class>
 <widget class="QDialog" name="VnosObveznosti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>439</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>647</width>
        <height>439</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>VNOS OBVEZNOSTI</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Listina</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_listina</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_listina">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Dobavitelj</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_dobavitelj</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_dobavitelj">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Znesek</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_znesek</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_znesek">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Datum valute</string>
            </property>
            <property name="buddy">
             <cstring>dateEdit_valute</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_valute">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Opombe</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_opombe</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_opombe">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_vnos">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Vnos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_popraviObv">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Popravi obveznost</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_izhod">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Izhod</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_listina</tabstop>
  <tabstop>lineEdit_dobavitelj</tabstop>
  <tabstop>lineEdit_znesek</tabstop>
  <tabstop>dateEdit_valute</tabstop>
  <tabstop>lineEdit_opombe</tabstop>
  <tabstop>pushButton_vnos</tabstop>
  <tabstop>pushButton_izhod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_izhod</sender>
   <signal>clicked()</signal>
   <receiver>VnosObveznosti</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
